src/DatePicker/Date.elm

```diff
@@ -1,6 +1,11 @@
 module DatePicker.Date
     exposing
-        ( initDate
+        ( YearRange
+        , yearRangeActive
+        , between
```

Comment:
`DatePicker.Date` isn't exposed by this library. Won't that mean that no one can actually override `changeYear` in practice? If so, I'm not sure we should expose all of these exports just to get `YearRange` constructors out there. What do you think about re-exporting them from `DatePicker`?

(Personally, I have a bias toward over-exposing API elements, since Elm makes major version upgrades so aggressively anyway, but I inherited the repo and am trying to respect its design thus far.)